test/hotspot/jtreg/compiler/loopopts/superword/TestCompatibleUseDefTypeSize.java

```diff
@@ -327,12 +327,12 @@ static Object[] test0(byte[] src, char[] dst) {
     }
 
     @Test
-    @IR(counts = {IRNode.STORE_VECTOR, "= 0"},
+    @IR(counts = {IRNode.STORE_VECTOR, "> 0"},
         applyIfPlatform = {"64-bit", "true"},
-        applyIfCPUFeatureOr = {"sse4.1", "true", "asimd", "true"})
+        applyIf = {"AlignVector", "false"},
+        applyIfCPUFeature = {"avx", "true"})
```

Comment:
This may be a little nit-picky. But why have a new test-file when this test here was already trying to cover the conversion cases? I think I wrote it back then, and was just too lazy to write all conversion cases. I'd suggest you move your cases up here ;)

Reply:
I think I added these tests when I was reworking `SuperWord::is_velt_basic_type_compatible_use_def`, which you are now touching as well :blush:

Reply:
I think I didn't realize this test was there when I was working on my patch, but I agree it makes sense to move it there for clarity.